lib/commands/init.js

```diff
@@ -44,19 +44,23 @@ async function run(options) {
         return
     }
     log(chalk.bold('Building packages'))
-    try {
-        await runner(
-            'forge-ui-components',
-            `${NPM} run build`,
-            { cwd: path.join(options.packageDir, 'forge-ui-components') }
-        )
-        log(`${chalk.greenBright('+')} flowforge/forge-ui-components`)
-    } catch(err) {
-        log(`${chalk.greenBright('-')} flowforge/forge-ui-components`)
-        error('Failed to build forge-ui-components')
-        error(err.stderr)
-        error('Aborting')
-        return
+    const buildPackages = ['forge-ui-components','flowforge']
+    for (let i = 0; i < buildPackages.length; i++) {
+        const package = buildPackages[i]
+        try {
+            await runner(
+                package,
+                `${NPM} run build`,
+                { cwd: path.join(options.packageDir, package) }
+            )
+            log(`${chalk.greenBright('+')} flowforge/${package}`)
+        } catch(err) {
+            log(`${chalk.greenBright('-')} flowforge/${package}`)
+            error(`Failed to build ${package}`)
+            error(err.stderr)
+            error('Aborting')
+            return
```

Comment:
JS noob again, but why `return` here? And not just letting the function exit as is, given there's no more statements in it?